src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -148,6 +149,8 @@ public Object instanceMainPostParseArgs() {
             BlockGunzipper.setDefaultInflaterFactory(new IntelInflaterFactory());
         }
 
+        BucketUtils.setGlobalNIODefaultOptions();
```

Comment:
I think we are going to want to expose the number of retries as an argument (instead of hardcoding it, as it currently is). This means that we have to make this call after arg parsing, rather than before.

When we reach a point where barclay has a need for GCS NIO support, we can revisit at that time.